elements/directory/xvocabulary/nuxeo-xvocabulary-edit-layout.html

```diff
@@ -38,7 +38,7 @@
       required$="[[_isParentRequired(parentDirectory, directory)]]"
     >
       <template is="dom-repeat" items="[[parentResponse.entries]]">
-        <paper-item name="[[item.properties.id]]">[[item.properties.id]]</paper-item>
+        <paper-item name="[[item.properties.id]]">[[_computeLabel(item)]]</paper-item>
```

Comment:
`_computeLabel` as a dependency on `this.i18n`, which in theory can change, so we would like the label to recompute if this happens:
```suggestion
        <paper-item name="[[item.properties.id]]">[[_computeLabel(item, i18n)]]</paper-item>
```